pmoves/services/tensorzero-config-api/logging.py

```
@@ -286,7 +286,7 @@ def log_configuration_update(
         )
 
         record = ConfigChangeRecord(
-            timestamp=datetime.utcnow(),
+            timestamp=datetime.now(timezone.utc),
```

Ruff (0.14.10)

Undefined name `timezone`

(F821)